cernopendata/modules/theme/assets/semantic-ui/scss/visualise.scss

```diff
@@ -28,6 +28,29 @@
   }
 }
 
+#parameter-button-row .item {
+  margin-left: 0.5em;
+}
+
+#vis-histograms .plot-control {
+  margin-left: 2em;
+}
+
+#vis-histograms .ui.button {
+  padding: 0.8em;
+  margin: 0;
+  font-weight: 500;
+}
+
+#vis-histograms .ui.action.input {
+  margin-left: 1em;
+  max-width: 150px;
+}
+
+#vis-histograms .undoselect {
```

Comment:
You can move this up under `#vis-histograms .ui-button` and thus you avoid the `!important`:

```css
#vis-histograms .ui.button {
  padding: 0.8em;
  margin: 0;
  font-weight: 500;

  &.undoselect {
    margin-left: 4em;
  }
}
```